python2verilog/api/verilogify.py

```diff
@@ -158,4 +159,22 @@ def get_actual(
             timeout=timeout,
         )
         assert not err, f"{stdout} {err}"
-        yield from strip_signals(parse_stdout(stdout), include_invalid=include_invalid)
+        yield from parse_stdout(stdout)
+
+
+def get_actual(
+    verilogified: FunctionType,
+    module: str,
+    testbench: str,
+    timeout: Optional[int] = None,
+) -> Iterator[Union[tuple[int, ...], int]]:
+    """
+    Get actual output of the testbench with rows
+
+    filtered by ready and valid signals,
+
+    and the signals themselves removed.
+    """
+    yield from strip_valid(
```

Comment:
If these functions throw an exception, shouldn't you catch them somewhere? Or are they caught somewhere else?

Reply:
I would expect the caller to catch them.